### PR TITLE
Support fling with gesture margin.

### PR DESCRIPTION
**Pull request by TranquilRock**
This change is [Reviewable](https://reviewable.io/reviews/google/snippet-uiautomator/61)

### CODE REVIEW

File: snippet_uiautomator/uiobject2.py [hunk 1]
```diff
@@ -165,38 +165,25 @@ def to(
...
       ui: snippet_client_v2.SnippetClientV2,
       selector: byselector.BySelector,
-      action: str,
   ) -> None:
     self._ui = ui
     self._device = self._ui._device  # pylint: disable=protected-access
     self._selector = selector
-    self._action = action
 
   def _perform_gesture(
```
Comment (ko1in1u) [inline snippet_uiautomator/uiobject2.py:180]: Rename to `_perform_swipe`?

Comment (TranquilRock) [reply snippet_uiautomator/uiobject2.py]: Sure, commit added.

**Review by ko1in1u — APPROVED** [review]

File: snippet_uiautomator/uiobject2.py (hunk 1 above)
Comment (TranquilRock) [reply snippet_uiautomator/uiobject2.py]: Done

**Review by TranquilRock — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/google/snippet-uiautomator/61)* status: 0 of 3 files reviewed, 1 unresolved discussion (waiting on @ko1in1u)

**Comment by ko1in1u** [comment]
Per offline discussed, this will break internal presubmit. I'll create another PR and fine tune the Python code to match with the current style.

**Pull request closed without merging**